File: OneDrive/Documentos/Store_Herencia/Store_Herencia/src/store/herencia/Food.java
package store.herencia;


public class Food extends Product{
    private String sell_by_date;
    public Food(String name, double price, String importDate, int stock, String sell_by_date) {
        super(name, price, importDate, stock);
        this.sell_by_date = sell_by_date;
    }

   @Override
    public String toString() {
        return super.getData() + String.format(", sell-by-date: %s", sell_by_date);
    }
}

File: PycharmProjects/Store_Herencia/py/Food.py
from Product import Product

class Food(Product):
    def __init__(self, name, price, import_date, stock, sell_by_date):
        super().__init__(name, price, import_date, stock)
        self.sell_by_date = sell_by_date

    def __str__(self):
        return super().get_data() + f", sell-by-date: {self.sell_by_date}"